webpack/assets/javascripts/react_app/redux/API/APIReducer.js

```diff
@@ -36,6 +36,15 @@ const apiReducer = (state = initialState, { type, key, payload, response }) => {
           status: ERROR,
         },
       });
+    case UPDATE:
+      return state.merge(
+        {
+          [key]: {
+            response: payload,
+          },
+        },
+        { deep: true }
```

Comment:
should we still use `deep` now that we pass the `prevState` and expecting a modified State?

Reply:
yes, now we don't need this "insurance"